src/common/utility.js

```diff
@@ -244,3 +244,11 @@ export const redirectOpenLiveChatBox = (is_redirect) => {
         navigate(live_chat_redirection_link, { replace: true })
     }
 }
+
+// Function which returns sub path to the specific trading platform
+const supported_platforms = ['mt5', 'bot', 'derivx']
+export const redirectToTradingPlatform = () =>
+    supported_platforms.filter(
+        (platform, index) =>
+            window.location.pathname.includes(platform) && supported_platforms[index],
+    )
```

Comment:
```suggestion
// Function which returns sub path to the specific trading platform
const supported_platforms = ['mt5', 'bot', 'derivx']
export const redirectToTradingPlatform = () =>
    supported_platforms.filter(
        (platform) =>
            window.location.pathname.includes(platform) && platform,
    )
```

Reply:
LGTM 👍  But in this case, you could just return the platform that is being looped through.

Reply:
Yeah, that's correct!